test/Deck-test.js

```diff
@@ -6,31 +6,30 @@ const Deck = require('../src/Deck');
 
 describe('Deck', function () {
 
-    it('should be a function', function () {
-        const deck = new Deck();
-        expect(Deck).to.be.a('function');
-    });
+  it('should be a function', function () {
+    const deck = new Deck();
```

Comment:
Do you need line 10 since your expect statement is only looking at "Deck" not "deck"?